src/ai/backend/client/func/keypair_resource_policy.py

```diff
@@ -54,7 +54,7 @@ async def create(
         max_vfolder_count: int,
         max_vfolder_size: int,
         idle_timeout: int,
-        allowed_vfolder_hosts: Sequence[str],
+        allowed_vfolder_hosts: str = None,
         fields: Iterable[FieldSpec | str] = None,
```

Comment:
Btw, as long as I know,  both`total_resource_slots` and `allowed_vfodler_hosts` have the same type (stringified JSON). Therefore, the type definition for `total_resource_slots` needs to be changed likewise.

Reply:
I read how keypair_resource_policy mutation works and it seems `total_resource_slots` should not be `None` because the field determine the policy. but I guess `total_resource_slots` can be `None` so I leave it Optional.

Reply:
I get it now. It should be left as is.